ui/apps/platform/src/services/CollectionsService.ts

```diff
@@ -216,13 +216,13 @@ export function getCollectionAutoComplete(
     searchCategory: SelectorField,
     searchLabel: string
 ): CancellableRequest<string[]> {
-    const params = qs.stringify(
-        { resourceSelectors, searchCategory, searchLabel },
-        { arrayFormat: 'repeat' }
-    );
     return makeCancellableAxiosRequest((signal) =>
         axios
-            .get<{ values: string[] }>(`${collectionsAutocompleteUrl}?${params}`, { signal })
+            .post<{ values: string[] }>(
+                `${collectionsAutocompleteUrl}`,
```

Comment:
Not a blocker to merge, does change in params let you replace template literal with variable?

Reply:
+1 Hmm it seems like this is something prettier would automatically fix. I'll push an update.